mtcpclient/calculate_test.go

```diff
@@ -6,6 +6,67 @@ import (
 	"time"
 )
 
+func TestCalculateMetricsReport(t *testing.T) {
+	var metricsReportScenariosChecks = []struct {
+		scenarioDescription         string
+		groupOfConnectionsToReport  GroupOfConnections
+		tcpStatusToReport           tcpclient.ConnectionStatus
+		expectedReportWithoutStdDev metricsCollectionStats
+	}{
+		{
+			scenarioDescription:        "Empty group of connections should report 0 as associated metrics",
+			groupOfConnectionsToReport: GroupOfConnections{},
+			tcpStatusToReport:          tcpclient.ConnectionEstablished,
+			expectedReportWithoutStdDev: metricsCollectionStats{
```

Comment:
why are you excluding StdDev?

Reply:
Just because its not responsibility of this test to check the result of the stddev calculation... we have an specific test for that